### PR TITLE
Update Extensions.swift

### PR DESCRIPTION
**Pull request by nuomi1**

### CODE REVIEW

File: Sources/Extensions.swift [hunk 1]
```diff
@@ -114,11 +102,11 @@ extension Bundle {
 extension String {
 
     var monkeyking_base64EncodedString: String? {
-        return data(using: .utf8)?.base64EncodedString(options: NSData.Base64EncodingOptions(rawValue: 0))
+        return data(using: .utf8)?.base64EncodedString(options: .lineLength64Characters)
```
[the remark this answers is not in the document]
Comment (nuomi1) [reply Sources/Extensions.swift]: It isn't... I'm sorry.

But I found [tenten0213](https://qiita.com/tenten0213/items/6ac8682e3a7e3b378167) recommended using `[]` instand of `(rawValue: 0)`.

File: Sources/Extensions.swift [hunk 2]
```diff
@@ -130,10 +118,8 @@ extension String {
     }
 
     var monkeyking_qqCallbackName: String {
-        var hexString = String(format: "%02llx", (self as NSString).longLongValue)
-        while hexString.count < 8 {
-            hexString = "0" + hexString
-        }
+        let hexString = String(format: "%08llx", (self as NSString).longLongValue)
```
[the remark this answers is not in the document]
Comment (nuomi1) [reply Sources/Extensions.swift]: That isn't working.

File: Sources/Extensions.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (nixzhu) [reply Sources/Extensions.swift]: OK, could you add a new commit to fix it?

**Comment by nixzhu** [comment]
@nuomi1 Thanks!

**Pull request merged**